core/templates/pages/exploration-player-page/learner-experience/input-response-pair.component.spec.ts

```diff
@@ -238,4 +238,15 @@ describe('InputResponsePairComponent', () => {
       ExplorationPlayerConstants.AUDIO_HIGHLIGHT_CSS_CLASS
     );
   });
+
+  it('should get the css class for feedback audio highlight', () => {
+    spyOn(audioTranslationManagerService, 'getCurrentComponentName')
+      .and.returnValue(AppConstants.COMPONENT_NAME_FEEDBACK);
+    spyOn(audioPlayerService, 'isPlaying').and.returnValue(false);
+    component.isLastPair = true;
+
+    expect(component.getFeedbackAudioHighlightClass()).toBe(
+      ''
+    );
```

Comment:
The test name doesn't properly explain what it does.

Reply:
Yes, I named it in a better way now.